main.go

```diff
@@ -1,7 +1,60 @@
 package main
 
-import "github.com/odpf/compass/cli"
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/odpf/compass/cli"
+)
+
+const (
+	exitOK    = 0
+	exitError = 1
+)
 
 func main() {
-	cli.Execute()
+	cliConfig, err := cli.LoadConfig()
+	if err != nil {
+		cliConfig = &cli.Config{}
+	}
+
+	rootCmd := cli.New(cliConfig)
+
+	if cmd, err := rootCmd.ExecuteC(); err != nil {
+		printError(err)
+
+		cmdErr := strings.HasPrefix(err.Error(), "unknown command")
+		flagErr := strings.HasPrefix(err.Error(), "unknown flag")
+		sflagErr := strings.HasPrefix(err.Error(), "unknown shorthand flag")
+
+		if cmdErr || flagErr || sflagErr {
```

Comment:
Can you double-check the handling for this? Usage is being printed twice currently:

```
$ ./compass config --test
Usage:  compass config <command> [flags]

Available commands:
  init
  list

unknown flag: --test

Usage:  compass config <command> [flags]

Available commands:
  init
  list
```